src/util.js

```diff
@@ -775,25 +775,26 @@ Oskari.util = (function () {
     /**
      * Format timestamp to more readable date
      * @param {String} text
-     * @param {Object} options optional
+     * @param {Object} options optional { time, date }
      * @param {Array || String} locales optional
      * @returns {String}
      */
-    util.formatDate = (text, options = {}, locales = []) => {
+    util.formatDate = (text, options, locales = []) => {
```

Comment:
Should we default locales to `Oskari.getSupportedLocales().filter(loc => loc.includes(Oskari.getLang()))`?